cdk/lib/bastion.ts

```diff
@@ -1,34 +1,32 @@
-import { Vpc, SecurityGroup, BastionHostLinux, SubnetType, Peer, Port, LookupMachineImage, MachineImage } from '@aws-cdk/aws-ec2';
-import { Construct, Stack, CfnOutput } from '@aws-cdk/core';
+import { aws_ec2 as ec2, Stack, CfnOutput } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
 
 const DEFAULT_AMI_MAP = {
   'us-east-1': 'ami-0c2b8ca1dad447f8a',
 };
 
 export interface CacclSshBastionProps {
-  vpc: Vpc,
-  sg: SecurityGroup;
-  amiMap?: { [key: string]: string },
-};
+  vpc: ec2.Vpc;
+  sg: ec2.SecurityGroup;
+  amiMap?: { [key: string]: string };
+}
 
 export class CacclSshBastion extends Construct {
-  instance: BastionHostLinux;
+  instance: ec2.BastionHostLinux;
 
   constructor(scope: Construct, id: string, props: CacclSshBastionProps) {
     super(scope, id);
 
-    const {
-      vpc,
-      sg,
-      amiMap = {},
-    } = props;
+    const { vpc, sg, amiMap = {} } = props;
 
-    this.instance = new BastionHostLinux(this, 'SshBastionHost', {
+    this.instance = new ec2.BastionHostLinux(this, 'SshBastionHost', {
```

Comment:
Not sure what rules are here, but I usually go by the "if one item is on a line, all the others are on their own lines too" rule...so something like:

```
myFunction(
  firstArg,
  secondArg,
  thirdArg,
)
```

Instead of

```
myFunction(firstArg, secondArg,
  thirdArg,
)
```